aexpect/ops_linux.py

```diff
@@ -297,8 +301,11 @@ def copy(session, source, target, quote_path=True, flags=""):
     Calls `cp source target` through a session. See `man cp` for what source
     and target can be and what behavior to expect.
     """
-    cmd = f'cp {flags} {quote(source)} {quote(target)}' if quote_path \
-        else f'cp {flags} {source} {target}'
+    cmd = (
+        f"cp {flags} {quote(source)} {quote(target)}"
+        if quote_path
+        else f"cp {flags} {source} {target}"
+    )
```

Comment:
Well this is a bit iffy, would you mind changing it to `if/then/else` instead?

Reply:
I see, ok will adapt it manually then.